core/netparams/keys.go

```diff
@@ -173,6 +173,7 @@ const (
 	SpamProtectionMaxApplyReferralCode     = "spam.protection.max.applyReferralCode"
 	SpamProtectionBalanceSnapshotFrequency = "spam.protection.balanceSnapshotFrequency"
 	SpamProtectionApplyReferralMinFunds    = "spam.protection.applyReferral.min.funds"
+	SpamProtectionReferralProgramMinFunds  = "spam.protection.referralProgram.min.funds"
```

Comment:
It probably wants to be `minFunds` but given the parameter above (line 175) it's probably too late for that.

Reply:
yeah.